src/main/java/com/hackclub/hccore/advancements/MillionAdv.java

```diff
@@ -0,0 +1,23 @@
+package com.hackclub.hccore.advancements;
+
+import com.fren_gor.ultimateAdvancementAPI.advancement.Advancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.BaseAdvancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplay;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplayBuilder;
+import com.fren_gor.ultimateAdvancementAPI.util.AdvancementKey;
+import com.fren_gor.ultimateAdvancementAPI.util.CoordAdapter;
+import com.hackclub.hccore.HCCorePlugin;
+import org.bukkit.Material;
+
+public class MillionAdv extends BaseAdvancement {
```

Comment:
why is this not automatically granted
PlayerStatisticIncrementEvent with FLY_ONE_CM -> convert to miles

Reply:
I'm looking at this right now, but I'm not sure if this actually works. The statistics are stored as `int`s, but from what I can see, 1 million miles in centimeters is above the integer limit.

Reply:
i agree with your observation, i was going off of the previous implementation.
does this mean it isnt possible to track?
https://github.com/hackclub/HCCore/blob/master/src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java#L103-L106

Reply:
Well I wonder what it returns when the player flys for more than the integer limit. I'll try to test that

Reply:
no need: https://bugs.mojang.com/browse/MC-176901
it just freezes (in vanilla)

Reply:
oh, ok. so should we decrease the miles needed to get the advancement or just delete it altogether?

Reply:
could decrease it to 500 miles (or 1000 for there and back)
![image](https://user-images.githubusercontent.com/42748092/222622955-c6104e18-317d-415b-9586-1acf67bd8ec6.png)